seatunnel-api/src/main/java/org/apache/seatunnel/api/configuration/util/RequiredOption.java

```diff
@@ -159,4 +159,45 @@ public String toString() {
             return String.format("Condition expression: %s, Required options: %s", expression, ExclusiveRequiredOptions.getOptionKeys(requiredOption));
         }
     }
+
+    /**
+     * All options must exist or not exist at the same time
+     */
+    class BothRequiredOptions implements RequiredOption {
```

Comment:
```suggestion
    class BundledRequiredOptions implements RequiredOption {
```

Reply:
IMO, the word `Bundled` is more appropriate than `Both`.

Reply:
+1